docs/pages/access-controls/access-graph/gitlab.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: Discover GitLab Access Patterns with Teleport Policy
+description: Describes how to syncronize GitLab access patterns using Teleport Policy and Access Graph.
+---
+Teleport Policy streamlines and centralizes access management across your entire infrastructure, allowing you to view
+access relationships within seconds. You can see unified, up-to-date relationships and policies between all users, groups,
+and computing resources.
+
+With Teleport Policy's Access Graph, you gain insights into access patterns within your GitLab account. By scanning all
+permissions, users, groups, and projects, it provides a visual representation to help enhance the permission model within
+your GitLab environment. This functionality enables you to answer queries such as:
+
+- What projects are accessible to users?
+- Which users have write permissions to projects?
+
+Access Graph is a feature of the [Teleport Policy](https://goteleport.com/platform/policy/) product
+available to Teleport Enterprise customers.
```

Comment:
```suggestion
available to Teleport Enterprise and Teleport Cloud customers.
```

Reply:
Our docs linter denies using `Teleport Enterprise and Teleport Cloud`, instead it suggests `Teleport Enterprise edition`